src/graphql/schemas/custom/patient.graphql

```diff
@@ -10,5 +10,5 @@ extend type Patient {
 }
 
 type Mutation {
-    updateGeneralPractitioner(patientId: String, practitionerId: String): Patient
+    updateGeneralPractitioner(patientId: String, practitionerId: String, remove: Boolean): Patient
```

Comment:
we tend to add docs for every graphql call to make it easier for callers to understand.  e.g, it is not clear to callers what "remove" means.  Another question is whether remove should be a separate mutation than update rather than add a flag?

Reply:
this was a request from the UI to make it one call so it can be used as a toggle.